Gems/Pointcloud/Code/Source/Tools/Builders/PointcloudBuilder.cpp

```diff
@@ -82,9 +82,21 @@ namespace Pointcloud
     {
         response.m_resultCode = AssetBuilderSDK::ProcessJobResult_Failed;
         happly::PLYData plyIn(request.m_fullPath.c_str());
-        auto vertices = plyIn.getVertexPositions();
-        auto colors = plyIn.getVertexColors();
         const auto elementNames = plyIn.getElementNames();
+        auto vertices = plyIn.getVertexPositions();
+        bool hasRed = AZStd::find(elementNames.begin(), elementNames.end(), "red") != elementNames.end();
```

Comment:
I would rather use std::find, since std::vector<std::string> is used as elementNames.